tests/unit/controllers/userRepoControllerTest.js

```diff
@@ -94,26 +94,20 @@ describe("controller: UserRepoController", function () {
   });
 
   describe("Does the scope method", function () {
-    it("allowableRoles return a list of allowed roles", function () {
-      var roles;
-
-      roles = $scope.allowableRoles();
-      expect(roles).toBeDefined();
-
-      initializeController({role: "ROLE_MANAGER"});
-
-      roles = $scope.allowableRoles("ROLE_ADMIN");
-      expect(roles).toBeDefined();
-
-      roles = $scope.allowableRoles();
-      expect(roles).toBeDefined();
-
-      roles = $scope.allowableRoles("ROLE_USER");
-      expect(roles).toBeDefined();
-
-      initializeController({role: "ROLE_USER"});
-      expect($scope.allowableRoles).not.toBeDefined();
-    });
+    // it("allowableRoles return a list of allowed roles", function () {
```

Comment:
At this stage in the game I would prefer to just delete the unused tests.

Reply:
Sure